include/rviz_visual_tools/rviz_visual_tools.h

```diff
@@ -1052,6 +1053,13 @@ class RvizVisualTools
   static float fRand(float min, float max);
   static int iRand(int min, int max);
 
+  /**
+   * \brief Set seed for random methods above
+	*
+	* Random sampling uses std::mt19937 internally
+	*/
```

Comment:
there is something weird going on with the formatting of the spacing in this doxygen comment

Reply:
it took 2 minutes to fix by smartphone in the web gui. please consider doing it during the review next time 🐶

Reply:
I had no idea you could edit a PR that way.  I always forget I can just fix things like this.

Reply:
Oh. Didn't know it was good etiquette to edit other people's PR's. Noted.

Reply:
> Oh. Didn't know it was good etiquette to edit other people's PR's

it's definitely an option when (1) the change has an obvious flaw the author failed to notice, (2) it can be squash-merged later, and (3) the maintainers are allowed to push to the pull-request branch. it simply wastes less time on trivia.

Reply:
Ah. I probably can't do that then